internal/goldendataset/span_generator.go

```diff
@@ -134,7 +134,7 @@ func generateSpanName(spanInputs *PICTSpanInputs) string {
 //  random - the random number generator to use in generating ID values
 //
 //The generated span is returned.
-func GenerateSpan(traceID []byte, parentID []byte, spanName string, spanInputs *PICTSpanInputs,
+func GenerateSpan(traceID otlpcommon.TraceID, parentID []byte, spanName string, spanInputs *PICTSpanInputs,
```

Comment:
You should use `pdata.TraceID` everywhere except in the declaration of the pdata.TraceID